torchao/quantization/quantize_/common/kernel_preference.py

```diff
@@ -30,5 +30,9 @@ class KernelPreference(str, Enum):
     """
     FBGEMM = "fbgemm"
 
+    """Use triton quantize and quantized mm kernels (if available), requires fbgemm_gpu_genai library, if no triton kernel for the quantize op or mm kernel is available, we'll fallback to torch ops
+    """
+    TRITON = "triton"
```

Comment:
this name isn't coherent with the rest of the enum.  We already have an `FBGEMM` option which does not say anything about cutlass vs triton and therefore already includes these kernels.  I think you have two options:
1. have the fbgemm option pick the best kernel (cutlass vs triton) for the user.  I prefer this one.
2. make it clear that "FBGEMM" does not mean "FBGEMM", but really means "FBGEMM_CUTLASS", and also add "FBGEMM_TRITON".  I don't really like this option.

Reply:
OK thanks, yeah 1 seems easiest for now, will update to that. unless there is request to distinguish these in the future